easybuild/easyconfigs/p/PHYLIP/PHYLIP-3.697-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'PHYLIP'
+version = '3.697'
+
+homepage = 'https://evolution.genetics.washington.edu/phylip'
+description = "PHYLIP is a free package of programs for inferring phylogenies."
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['https://evolution.gs.washington.edu/phylip/download/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['9a26d8b08b8afea7f708509ef41df484003101eaf4beceb5cf7851eb940510c1']
+
+builddependencies = [('binutils', '2.34')]
+
+buildopts = '-Csrc -fMakefile.unx all && make -Csrc -fMakefile.unx install'
+
+files_to_copy = [(['exe/*'], 'bin'), (['src/libdrawgram.so', 'src/libdrawtree.so'], 'lib')]
```

Comment:
The `.so` should be `SHLIB_EXT`. So these would change to be: `'src/libdrawgram.%s' % SHLIB_EXT` and `'src/libdrawtree.%s' % SHLIB_EXT`.